### PR TITLE
Doc 12259 quick fixes

### PR DESCRIPTION
**Pull request by ElliotFrancisHunter**
Quick fixes due to incorrect link pathing and overrides breaking some links due to '.3' 

Summary
-- Correct pathing
-- Fix broken links due to overrides
-- add downloads to what's new

### CODE REVIEW

File: modules/c/pages/gs-install.adoc [hunk 1]
```diff
@@ -85,7 +85,7 @@ For Android and iOS the symbols are incorporated in the standard release package
 
 You can obtain the download for the Vector Search extension here:
 
-* _Vector Search Extension_ -- link:https://www.couchbase.com/downloads/?family=couchbase-lite#LiteExtensions[Couchbase Lite Extensions]
+* _Vector Search Extension_ -- xref:gs-downloads.adoc#vs-release-1-0-0-beta.3[Download Vector Search]
```
Comment (osfameron) [inline modules/c/pages/gs-install.adoc:88]: hardcoded version?

Comment (osfameron) [reply modules/c/pages/gs-install.adoc:88]: ah, and hardcoded links throughout this file?
I guess this is a Quick Fix to tidy later?

**Review by osfameron — APPROVED** [review]
looks ok, if merging as is, then make a note to come back and tidy up the hardcoded version links?

File: modules/c/pages/gs-install.adoc (hunk 1 above)
Comment (ElliotFrancisHunter) [reply modules/c/pages/gs-install.adoc:88]: Previous link pointed to couchbase's download page rather than our doc download page... confusing I know

Comment (ElliotFrancisHunter) [reply modules/c/pages/gs-install.adoc:88]: Ah misunderstood. Quick fix to get it working correctly. Now correcting this and some other minor formatting issues in a separate PR

**Pull request merged**